CMake/scipp-util.cmake

```diff
@@ -47,11 +53,11 @@ function(scipp_function template category function_name)
   set(python_binders_fwd python_${category}_binders_fwd)
   set(python_binders python_${category}_binders)
   set(${python_binders_fwd}
-      "${${python_binders_fwd}}\nvoid init_${NAME}(pybind11::module &)ENDL"
+      "${${python_binders_fwd}}\nvoid init_${OPNAME}(pybind11::module &)ENDL"
```

Comment:
Can't you just escape the semi-colon `\;`? rather than doing ENDL and later replace?

Reply:
I think I had tried that, but cmake and strings...